src/mappings/generatedColumns.ts

```diff
@@ -39,12 +39,12 @@ export const extractEventArgs = (args: any[]) => {
  * Function to get value for a specific key
  * It searches for snake_case and camelCase value for the given key
  */
-export const extractValue = (obj: unknown, key: string): string => {
+export const extractValue = <T>(obj: unknown, key: string): T => {
```

Comment:
(style) What about defining some wrapper functions that will define the type for the user.

```
export const extractString = (obj: unknown, key: string) => extractValue<string>(obj, key);
```

I think `extractString` is easier to read than `extractValue<string>`. If its a one off thing then its OK to leave as the generic call, but for the common ones I think helpers could be nice, like `string` and `number`.